GMCMultiImage/GMCZoomingMultiImageView.m

```diff
@@ -212,29 +212,31 @@ - (void)updateImage {
                     }
                     
                     if (!error) {
-                        dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_LOW, 0), ^{
+                        dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
I thought it would be as well, but it wasn't (at least in the demo app). Looking up a few lines, this is actually the completion block of the fetch. It would probably be somewhat valid to expect the subclasses to dispatch to the main thread when performing the completion block, but I personally don't like forcing users to know about those assumptions (especially since the demo project doesn't even do that), so I'd prefer to leave this here.